care/facility/models/patient_data.py

```diff
@@ -92,7 +92,9 @@ class PatientRegistration(models.Model):
     past_travel = models.BooleanField(
         default=False, verbose_name="Travelled to Any Foreign Countries in the last 28 Days"
     )
-    countries_travelled = models.TextField(default="", blank=True, verbose_name="Countries Patient has Travelled to")
+    countries_travelled = models.TextField(
+        default="", null=True, blank=True, verbose_name="Countries Patient has Travelled to"
```

Comment:
why is default value required, when you have already set `blank=True` for form and `null=True` for database.

Reply:
Thanks, I've removed the default field and updated the commit.

Reply:
I think you will have to migrate again for the change to be transferred to the database. I can't see the new migration file